README.md

```diff
@@ -35,38 +35,40 @@ npm install dialog-size
         (10rem huge)
         20rem;  // size(1)
     ```
-3. Call `size()` function
+3. Call the `size()` function
 
     ```scss
     body {
         padding: size(-1); // => 0.2rem
         font-size: size(0); // => 1rem
     }
+
     h1 {
         font-size: size(1); // => 20rem
-        padding: size(large) 0; // => 1.5rem
+        padding: size(large) 0; // => 1.5rem 0
     }
+
     p {
         padding: size(3rem); // => 3.5rem
     }
     ```
 
 ## Usage 📝
 
-Once done your $dialog-sizes is set up, the `size()` function can be used in different ways:
+Once done your $dialog-sizes is set up, the `size()` function can be used in three different ways
 
 ### Relative system
 
+- `size(0)` will return the base size (1rem)
 - `size(1)` will return the largest size in your system (20rem)
 - `size(-1)` will return the smallest size in your system (0.2rem)
```

Comment:
Just rearranged here. For me having the base zero value first is the most logical.